### ISSUE TITLE
Grain mill

### ISSUE THREAD

**Issue by Olechr04** [MapsOfNorway/rennebugs#56]
I cant tip grain at grain mill, i've buyed the mill but it didn't helped

**Comment by Olechr04** [comment]
> What type are you trying to sell? I delivered wheat there and I don't own the mill

I was trying to sell some rapeseed👍

**Comment by OlaHaldor** [comment]
I am looking into a solution for this ; I got access to a Mac for a couple days on vacation, so I can check the code.
I have indeed forgot to add a capacity setting for canola and soy. (or forgot to remove them entirely..) 

However, looking at the different reports from social media, YouTube and here, I'm turning my idea of "staying real to the location" and will enable ALL sorts of crops that can be produced into something. 

Grain for flour
Olive, sunflower and canola for oil
Grape for raisin and juice
Sugar beet for sugar

I hope this will be an OK solution for everyone. I'm at peace with this solution, especially because I no longer have to think about where to put the other specific buildings. I just can't imagine where they'd fit in, and what they'd look like (because the default buildings don't really fit the looks).
